lens-data/lens/applicable.rkt

```diff
@@ -3,7 +3,7 @@
 (require reprovide/reprovide)
 (reprovide "main.rkt")
 
-(require (only-in "private/base/base.rkt" use-applicable-lenses!))
+(require (only-in lens/private/base/base use-applicable-lenses!))
```

Comment:
Applicable lenses yes, but `lens/applicable` needs to be in the `lens-data` package. There could be another module `lens/common/applicable` to do this though.